.github/release-drafter.yml

```diff
@@ -0,0 +1,29 @@
+name-template: '$RESOLVED_VERSION'
+tag-template: '$RESOLVED_VERSION'
+categories:
+  - title: 'Breaking'
+    label: 'major'
+  - title: 'Added'
+    label: 'minor'
+  - title: 'Fixed'
+    label: 'patch'
+  - title: 'Dependency updates'
+    label: 'dependencies'
+exclude-labels:
+  - 'skip-changelog'
+change-template: '- $TITLE (#$NUMBER)'
+version-resolver:
+  major:
+    labels:
+      - 'major'
+  minor:
+    labels:
+      - 'minor'
+  patch:
+    labels:
+      - 'patch'
+      - 'dependencies'
+  default: patch
+template: |
```

Comment:
It might be good to add $CONTRIBUTORS to the template. It will be quite nice to mention the people who contributed to the change.

Reply:
Actually removed this again, but using the default template. I found if i added this it also added a second list of contributors below the changelog with images etc. Then I noticed that if I just use the default change template it includes the contributor in the change AND below (see https://github.com/marianeum/release-model-playground/releases)